pysteps/extrapolation/semilagrangian.py

```diff
@@ -80,6 +81,8 @@ def extrapolate(
     vel_timestep : float
         The time step of the velocity field. It is assumed to have the same
         unit as the timesteps argument.
+    interp_order : int
+        Order of the interpolation to use. Default : 0 (nearest interpolation).
```

Comment:
the documentation doesn't match the actual default value (`interp_order=3`)

Reply:
Fixed.